public/cli/lib/commands/convert.ts

```diff
@@ -56,3 +56,56 @@ export async function convert(
 
   return outputFilePath;
 }
+
+let lastCurrentFrame = 0;
```

Comment:
I'm not excited about the use of 2 global variables. That opens us up to bugs we could do without. However, I guess we can refactor that later, since this gets us going.